src/model.cpp

```diff
@@ -1571,7 +1571,6 @@ int Model::checkFinite(const int N, const realtype *array, const char *fun) cons
     auto result = amici::checkFinite(N, array, fun);
 
     if(result != AMICI_SUCCESS) {
-        amici::checkFinite(ts.size(), ts.data(), "ts");
```

Comment:
When using adjoints, a check for inf might be appropriate. Not sure if that is done in any other place

Reply:
hmm good point!